Lib/javascript/v8/javascripthelpers.swg

```diff
@@ -61,23 +61,24 @@ SWIGRUNTIME void SWIGV8_AddMemberVariable(SWIGV8_FUNCTION_TEMPLATE class_templ,
  * Registers a class method with given name for a given object.
  */
 SWIGRUNTIME void SWIGV8_AddStaticFunction(SWIGV8_OBJECT obj, const char* symbol,
-  const SwigV8FunctionCallback& _func) {
-#if (V8_MAJOR_VERSION-0) < 4 && (SWIG_V8_VERSION < 0x031903) || (SWIG_V8_VERSION < 0x0704)
+  const SwigV8FunctionCallback& _func, v8::Local<v8::Context> context) {
```

Comment:
IMHO, `context` should be passed as reference istead of value.

Reply:
v8::Local<T> is a pointer placeholder. In other words it's not v8::Context that is passed by value, but pointer/reference to v8::Context.